documentation/contributing.md

```diff
@@ -1,11 +1,26 @@
 # Contribution guidelines
 
-Contributions are welcome! In addition to core contributions, developers are encouraged to build their own custom transformers which
+Contributions are welcome! Please open an Issues or Pull Request for any changes.
+
+In addition to core contributions, developers are encouraged to build their own custom transformers which
 can be run together with other custom transformers using the [composeAndExeucte](../../staging/documentation/composeAndExecute.md) command.
 
+## Pull Request Process
+- Make sure the build is passing.
+- Update the README or any documentation files as necessary. If editing the Readme, please
+conform to the
+[standard-readme specification](https://github.com/RichardLitt/standard-readme).
+- You may merge a Pull Request once you have an approval from core developer.
```

Comment:
This is the general workflow that we've been following, but I wonder if we should rethink this in the future when additional contributors come onto the project so that we can continue to sanely add new features. Do folks think that we should require 2 approvals? Only allow "core developer" merging permissions?

Reply:
I wonder if we should also mention to tag a few of us as Reviewers on any new Pull Request?

Reply:
Definitely on board with requiring 2 approves and only allowing core members to merge PRs. I think (hope) the latter is already the case

Reply:
👍 cool, I'll update the language in this document, and change the repo settings to require 2 reviews before allowing a merge.